src/MySqlConnector/ColumnReaders/BinaryDateTimeColumnReader.cs

```diff
@@ -0,0 +1,74 @@
+namespace MySqlConnector.ColumnReaders;
+
+using System;
+using System.Buffers.Text;
+using System.Runtime.InteropServices;
+using System.Text;
+using MySqlConnector.Core;
+using MySqlConnector.Protocol.Payloads;
+using MySqlConnector.Protocol.Serialization;
+using MySqlConnector.Utilities;
+
+internal sealed class BinaryDateTimeColumnReader : IColumnReader
+{
+	private bool allowZeroDateTime;
+	private bool convertZeroDateTime;
+	private DateTimeKind dateTimeKind;
```

Comment:
Please follow the style of existing classes:

* fields are placed at the end of the class definition
* private fields are prefixed with `m_`

Additionally, these fields should be `readonly`.

Will fix in https://github.com/mysql-net/MySqlConnector/pull/1331/commits/10a3c08a9d7cbea5a2de1a57c6cb048cb96390d7.